src/scripts/main.js

```diff
@@ -1,3 +1,61 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+
+function sort(nameSort) {
+  const tbody = document.querySelector('tbody');
+  const tr = tbody.querySelectorAll('tr');
+  let n;
+
+  switch (nameSort) {
+    case 'Name':
+      n = 0;
+      break;
+    case 'Position':
+      n = 1;
+      break;
+    case 'Age':
+      n = 2;
+      break;
+    case 'Salary':
+      n = 3;
+      break;
+  }
+
+  const people = [...tr].sort((a, b) => {
+    if (n === 0 || n === 1) {
+      return (a.querySelectorAll('td')[n].textContent).localeCompare(
+        b.querySelectorAll('td')[n].textContent);
+    }
+
+    return (
+      a.querySelectorAll('td')[n].textContent.replace('$', '').replace(',', ''))
+    - b.querySelectorAll('td')[n].textContent.replace('$', '').replace(',', '');
+  });
```

Comment:
You don't need a `n` variable here. Imagine if you had a lot of possible sort types, what would you do? Just add a switch/case statement inside the sort function.